apps/ui/src/stores/votingPowers.ts

```diff
@@ -34,6 +35,13 @@ function isSpace(item: SpaceDetails | Proposal): item is Space {
   return 'proposal_threshold' in item;
 }
 
+function isSpaceMember(space: Space, account: string): boolean {
+  return [
+    ...(space.additionalRawData?.admins || []),
+    ...(space.additionalRawData?.moderators || [])
+  ].some(member => compareAddresses(member, account));
```

Comment:
maybe we should add `members` similar too https://github.com/snapshot-labs/snapshot-sequencer/blob/master/src/writer/proposal.ts#L117-L126 ? 🤔

Reply:
We already have validation for `onlyMembers`, via https://github.com/snapshot-labs/sx-monorepo/blob/ff5bf07bba44c47794cffbd165aaeb8fdf4e4ba9/packages/sx.js/src/strategies/offchain/only-members.ts#L5 , which allow members to create proposal if `onlyMember` is enabled in the space settings.

The current fix is about allowing team members to bypass all proposal validations, and is using the stores instead of sx.js, since this logic is not an existing proposalValidation. 

Moving this logic inside sx.js strategies will require more refactoring, as those strategies are not space aware.

Reply:
But the problem is that an author address (0x89446aF03652c5257dB5C8E4E85495EB754196c5) on space thanku.eth is able to create proposal on https://snapshot.org/#/thanku.eth but not on https://snapshot.box/#/s:thanku.eth

Reply:
where `onlyMembers` is not switched on, and author address doesn't have voting power

Reply:
> But the problem is that an author address (0x89446aF03652c5257dB5C8E4E85495EB754196c5) on space thanku.eth is able to create proposal on https://snapshot.org/#/thanku.eth but not on https://snapshot.box/#/s:thanku.eth

But your space does not have `onlyMember` enabled, and the address you gave is not an admin/moderator, and does not have any voting power. He should not be able to create proposals

Reply:
<img width="438" alt="Untitled 6" src="https://github.com/user-attachments/assets/ded935e2-91d5-4860-9f94-d610febdc56c">

But `author` (member) should be able to create proposals without any voting power/without `onlyMember` 😅

Reply:
It is quite confusing with all naming but it is written on sequencer 🙈 all members (admins/moderators/authors) should be able to create proposals irrespective of `onlyMembers` and proposal validation

Reply:
You're correct, I got confused